xoinvader/animation.py

```diff
@@ -91,14 +91,23 @@ def update(self):
 class Animation(object):
     """Animation unit.
 
+    Animation object holds sorted list of (time, value) items and changes
+    selected attribute of binded object according to local animation time.
```

Comment:
Yes. bind is a verb, and bound is it's perfect form, acting as adverb also. There's no such word as 'binded'.
http://en.bab.la/conjugation/english/bind